### ISSUE TITLE
Question: Is The Text Box Outline Superfluous?

### ISSUE THREAD

**Issue by qu0zz** [Ginj-capture/Ginj#65]
The text box outline is, in my opinion, very restrictive as currently it has to be the same colour as the text inside it.
Also a box outline may not be visually appropriate.
Since there is a separate box option I think the text box OUTLINE could be removed completely, with little or no detriment
That might make the implementation of text background colour (suggested in previous issue) a little easier? (I'm just guessing)

**Comment by vicnevicne** [comment]
Agreed that the text box outline is quite limited in options now, but I don't think removing the outline would be a good idea, at least by default. I, for one, really like the ability to highlight using text boxes without having to fiddle with controls, like:
![image](https://github.com/Ginj-capture/Ginj/assets/842233/e53808aa-e5b2-486c-8bfa-86a799e3473e)
But I'll try to add the options in the future :-)
Thanks for the feedback.
